omnibus/files/private-chef-cookbooks/private-chef/templates/default/nginx/scripts/resolver.lua.erb

```diff
@@ -43,10 +43,7 @@ upstream_resolver.tarpitify = function(route_id, org_config, internal)
 end
 
 upstream_resolver.acct_erchef_fun = function(route)
-  if route.org_config["couchdb_" .. route.endpoint] == 0 then
-    return "erchef"
-  end
-  return "acct"
+  return "erchef"
```

Comment:
To clarify - there's a fair bit that can be done to clean up the routing rules, while still capturin gorg/endpoint/object data for both darklaunch and future stats capture.  But I'd like to take a day to sit down with it since it'll be a fair refactor.  

I'll add a card for it, as well as a for a migration to clean up existing darklaunch keys.